core/src/main/scala/org/apache/spark/scheduler/cluster/SparkDeploySchedulerBackend.scala

```diff
@@ -61,8 +61,10 @@ private[spark] class SparkDeploySchedulerBackend(
       "--worker-url", "{{WORKER_URL}}")
     val extraJavaOpts = sc.conf.getOption("spark.executor.extraJavaOptions")
       .map(Utils.splitCommandString).getOrElse(Seq.empty)
-    val classPathEntries = sc.conf.getOption("spark.executor.extraClassPath")
-      .map(_.split(java.io.File.pathSeparator).toSeq).getOrElse(Nil)
+
+    val classPathEntries = sc.conf.getExecutorExtraClassPath().map(_.split(java.io.File
```

Comment:
Formatting is a little off. Can you split this into 2 lines a bit better? Or you could:
`val classPathEntries = 
  sc.conf.getExecutorExtraClassPath()
  .map(_.split(java.io.File.pathSeparator).toSeq).getOrElse(Nil)`